server/channels/jobs/base_workers_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package jobs
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSimpleWorkerPanic(t *testing.T) {
+	jobServer, mockStore, mockMetrics := makeJobServer(t)
```

Comment:
Should `makeJobServer` also assign a logger to the returned `JobServer`?

Reply:
I tried that before, but it was throwing some error. Let me try again.